package.json

```diff
@@ -18,68 +18,71 @@
 		"license.txt"
 	],
 	"scripts": {
-		"prebuild": "npm run install-if-deps-outdated",
-		"build": "npm run build:feature-config && npm run build:packages && cross-env NODE_ENV=production webpack",
+		"preinstall": "npx only-allow pnpm",
```

Comment:
I am not prevented from running `npm install`.

Local npm version is 8.1.0 if it matters.

Reply:
> I am not prevented from running npm install.
> Local npm version is 8.1.0 if it matters.

Yes, I was able to reproduce this with npm7 and npm8, and also able to see this when running `npm install` in woo core.

I found a relevant npm issue:
- https://github.com/npm/cli/issues/2660

Reply:
@jacob-sewell Currently, I think we don't have another way to prevent running `npm install` for `npm >= 7`. It would be great if you/or anyone has some idea. Otherwise, I would probably create another issue to follow up it. Thanks!